### PR TITLE
SAK-45092 Create a new tool for handling conversations in Sakai

### PR DESCRIPTION
**Pull request by adrianfish**
https://jira.sakaiproject.org/browse/SAK-45092

### CODE REVIEW

**Comment by mpellicer** [comment]
Only 310 lines removed? Where's forums deletion!

**Comment by adrianfish** [comment]
Codacy's loving this PR

File: webcomponents/tool/src/main/frontend/.storybook/main.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 module.exports = {
   "stories": [
     "../stories/**/*.stories.mdx",
+    "../stories/**/*.stories.@(js|jsx|ts|tsx)",
```
Comment (jesusmmp) [inline webcomponents/tool/src/main/frontend/.storybook/main.js:4]: Duplicated?

Comment (adrianfish) [reply webcomponents/tool/src/main/frontend/.storybook/main.js:4]: No, they're different suffixes, but they can be on one line, good spot!

Comment (adrianfish) [reply webcomponents/tool/src/main/frontend/.storybook/main.js:4]: We can get rid of the jsx, ts and tsx suffixes as we don't actually use those formats for stories. It was auto generated.

**Review by ern — APPROVED** [review]

**Pull request merged**